src/components/WorkStyleCard/index.jsx

```diff
@@ -1,67 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import cn from 'classnames';
-import { Transition } from 'react-spring/renderprops';
-import { animated } from 'react-spring';
+import { useTransition, animated } from 'react-spring';
 
-export default class WorkStyleCard extends React.Component {
-  constructor() {
-    super();
-    this.state = { isHovered: false };
-  }
-  render() {
-    const borderColor = {
-      red: 'border-c200',
-      green: 'border-c400',
-      yellow: 'border-c300',
-      blue: 'border-c500'
-    };
-    return (
-      <animated.div
-        className=" card__wrapper text-center flex flex-col col-div items-center my-8   md:my-auto"
-        onMouseEnter={() => this.setState({ isHovered: true })}
-        onMouseLeave={() => this.setState({ isHovered: false })}
-        style={this.props.fade}
+const WorkStyleCard = ({
+  description,
+  title,
+  img,
+  img_hover,
+  border_color,
+  animation
+}) => {
+  const [isHovered, setHoverState] = useState(false);
+  const transitions = useTransition(isHovered, null, {
+    from: { position: 'absolute', opacity: 0 },
+    enter: { opacity: 1 },
+    leave: { opacity: 0 },
+    config: { duration: 400 }
+  });
+  const borderColor = {
+    red: 'border-c200',
+    green: 'border-c400',
+    yellow: 'border-c300',
+    blue: 'border-c500'
+  };
+  return (
+    <animated.div
+      className=" card__wrapper text-center flex flex-col col-div items-center my-8  md:my-auto"
+      onMouseEnter={setHoverState(true)}
```

Comment:
these 2 lines are the cause of the issue but i'm not sure why?
if i remove them the problem is solved